sql/001_create_volunteers.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE `volunteers` (
+  `volunteer_id` int NOT NULL,
+  `volunteer_name` varchar(45) NOT NULL,
+  `hours_worked` int NOT NULL,
+  `ranking` int NOT NULL,
```

Comment:
IMO, ranking field can be dropped - as points as updated, you can recalculate ranking by using `select ... order by`.